vscode-powershell.build.ps1

```diff
@@ -145,15 +137,15 @@ task Package UpdateReadme, {
         throw "Unable to find PowerShell EditorServices"
     }
 
-    Write-Host "`n### Packaging PowerShell-insiders.vsix`n" -ForegroundColor Green
+    $packageName = "$($script:PackageJson.name)-$($script:PackageJson.version).vsix"
+    Write-Host "`n### Packaging $packageName`n" -ForegroundColor Green
     exec { & node ./node_modules/vsce/out/vsce package --no-gitHubIssueLinking }
 
-    # Change the package to have a static name for automation purposes
```

Comment:
@rjmholt You mentioned a while ago that there were a number of "automation purposes" for which this was named `PowerShell-Insiders.vsix`, but at this point, I don't think they're around any more. We've essentially rewritten the whole of the release process (and it's getting to be almost buttery smooth now). Am I missing anything?

Reply:
AFAICR I think we should be good